komodo-test-utils/src/test/resources/org/komodo/test/utils/test-local-repository-in-memory-config.json

```diff
@@ -6,13 +6,6 @@
     },
     "node-types" :
     [
-    	"config/dv.cnd",
-        "config/xmi.cnd",
-        "config/med.cnd",
-        "config/mmcore.cnd",
-        "config/relational.cnd",
-        "config/vdb.cnd",
-        "config/TeiidSql.cnd",
         "config/komodo.cnd"
```

Comment:
Not sure who you get your *suggestions* from, but teiid-modeshape uses all those CNDs and doesn't plan on removing any of them. So please don't worry, they are not going anywhere. What is brittle in my opinion, is having multiple copies of the same file in different projects when only one (the master copy) is needed.